tests/NLog.UnitTests/LayoutRenderers/AssemblyVersionTests.cs

```diff
@@ -45,6 +44,11 @@ public class AssemblyVersionTests : NLogTestBase
     {
         private readonly ITestOutputHelper _testOutputHelper;
 
+#if !NETSTANDARD
+        private static Assembly TestAssembly => _testAssembly ?? (_testAssembly = GenerateTestAssembly());
```

Comment:
Maybe we could use Lazy in the test project? :)

Reply:
Fixed